### PR TITLE
Fix-all-the-little-inconsistencies-in-the-json-command-file-descriptions

### PR DESCRIPTION
**Pull request by ItsAJ1005**
I am new to open-source contribution and have attempted to make some minor changes. I would greatly appreciate a review from your side. Thank you!

### CODE REVIEW

**Comment by ItsAJ1005** [comment]
If you like my work, please review!

File: src/commands/acl-dryrun.json [hunk 1]
```diff
@@ -23,7 +23,7 @@
                 },
                 {
                     "type": "string",
-                    "description": "The description of the problem, in case the user is not allowed to run the given command."
+                    "description": "The description of the problem in case the user is not allowed to run the given command."
```
Comment (PingXie) [inline src/commands/acl-dryrun.json:26]: should we replace "in case" with "if" here too?

File: src/commands/acl-genpass.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
         ],
         "reply_schema": {
             "type": "string",
-            "description": "Pseudorandom data. By default it contains 64 bytes, representing 256 bits of data. If `bits` was given, the output string length is the number of specified bits (rounded to the next multiple of 4) divided by 4."
+            "description": "Pseudorandom data. By default, it contains 64 bytes, representing 256 bits of data. If `bits` was given, the output string length is the number of specified bits (rounded to the next multiple of 4) divided by 4."
```
Comment (PingXie) [inline src/commands/acl-genpass.json:18]: should we remove "," in the phrase below too?

```
it contains 64 bytes, representing 256 bits of data
```

**Comment by PingXie** [comment]
Thanks @ItsAJ1005! Really appreciate the attention to details.

**Comment by zuiderkwast** [comment]
#63 is ready to merge. Just waiting for broken file names (redis-cli is used in a python script)

**Comment by zuiderkwast** [comment]
@ItsAJ1005 I merged #63. Sorry for causing you merge conflicts but that PR was already reviewed and ready to merge. I hope you can resolve them easily enough.

**Pull request closed without merging**